sdk/servicebus/service-bus/test/perf/service-bus-v1/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can we add "private": true to these package.json files just so we can't accidentally publish them?

Reply:
Oh nice